### PR TITLE
feat: slider component

### PR DESCRIPTION
**Pull request by Sameony**
feat: slider component
image
image

### CODE REVIEW

File: Templates/Slider/slider.html [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+
+            <datalist id="tickmarks" class="flex flex-spc-btw fs-sm">
+                <option value="0" class="fw-lg" label="0"></option>
+                <option value="2500"></option>
+                <option value="5000" class="fw-lg" label="5K"></option>
+                <option value="7500"></option>
+                <option value="10000" class="fw-lg" label="10K"></option>
+            </datalist>
+
+
```
Comment (tanishq20) [inline Templates/Slider/slider.html:172]: remove extra spaces.

**Review by tanishq20 — APPROVED** [review]
Very well-written clean code. Go ahead.

**Review by aishwarya-bm — APPROVED** [review]
Great usgae of utils, good to merge

**Review by vaibhav-mahalle — APPROVED** [review]
Everything looks good. just a suggestion try to change utilities in separate commit.

**Pull request merged**